Libraries/FBReactNativeSpec/FBReactNativeSpec.podspec

```diff
@@ -9,13 +9,20 @@ package = JSON.parse(File.read(File.join(__dir__, "..", "..", "package.json")))
 version = package['version']
 
 source = { :git => 'https://github.com/facebook/react-native.git' }
+codegenPathPrefix = ".."
 if version == '1000.0.0'
   # This is an unpublished version, use the latest commit hash of the react-native repo, which we’re presumably in.
   source[:commit] = `git rev-parse HEAD`.strip
+  codegenPathPrefix = "packages"
 else
   source[:tag] = "v#{version}"
 end
 
+reactNativePath = File.join(__dir__, "..", "..")
+codegenScriptPath = File.join(reactNativePath, "scripts", "generate_native_modules_specs.rb")
+codegenPath = File.join(reactNativePath, codegenPathPrefix, "react-native-codegen")
+codegenCommand = "CODEGEN_PATH=#{codegenPath} ruby '#{codegenScriptPath}'"
```

Comment:
A little context on what is going on here. We need to address two scenarios: a contributor using RNTester within a clone of the react-native repository, and a end-user developing an iOS app with React Native as a dependency.

In the former case, the codegen is available locally in the `react-native` repository checkout. In the latter, the codegen is installed from npm and is in node_modules. We already have some logic in place to determine which one is the case: the version will be set to 1000.0.0 when working inside the react-native repository, otherwise it will match the actual open source version number (e.g. 0.64.0).

In either case, the relative path from this podspec file to the React Native root is the same.